__tests__/app.test.js

```diff
@@ -18,19 +17,27 @@ describe('errors - 404', () => {
         .get('/api/notAValidPath')
         .expect(404)
         .then(({body}) => {
-            console.log(body.message);
             expect(body.message).toBe('No such path...\n...yet');
         })
     })
 })
-
 describe('GET /api/topics', () => {
     it('returns status 200 and the correct number of topic objects i.e. contains all the topics', () => {
         return request(app)
         .get('/api/topics')
         .expect(200)
-            .then(({ body }) => {
+        .then(({ body }) => {
             expect(body.message).toHaveLength(3)
         })
     })
+})
+describe('GET /api', () => {
+    it('return status 200 and the list of endpoints available', () => {
+        request(app)
+        .get('/api')
+        .expect(200)
+            .then(({ body }) => {
+                expect(body.message).toHaveLength(3)
```

Comment:
We also want to check the contents of what is coming back.
This can be done by asserting that what comes back from our request looks like our endpoints.json